examples/import_catalog_items/script.py

```diff
@@ -0,0 +1,223 @@
+import csv
+from dataclasses import dataclass
+import io
+import logging
+import typing as t
+
+import push_to_3yourmind
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+"""
+An example script that uses push_to_3yourmind to read out a csv
+and import the data to catalog items in an example platform.
+
+However, with providing real world config data and a custom
+mapper function this script can be used in a productio environment
+as well.
+"""
+
+
+@dataclass
+class Config:
```

Comment:
@HansAuger (1) since this is environment specific, it would be a bit nicer if those configs are externalised into a separate file...

Reply:
I think I will remove that. It made sense when in the dedicated package I added an optional cli, but here it is somewhat overengeneered